dj_project/mysite/goods/views.py

```diff
@@ -1,44 +1,12 @@
 from django.shortcuts import render
+from django.http import Http404
+from .models import Product, Category
 
 
-def homepage(request):
-    context = {
-        "urls": [
-            {
-                "model": "motorola-edge-30-pro",
-                "picture": "https://mobile-review.com/all/wp-content/uploads/2022/04/1-1.jpg"
-            },
-            {
-                "model": "samsung-galaxy-a53",
-                "picture": "https://i.allo.ua/media/catalog/product/cache/3/image/524x494/602f0fa2c1f0d1ba5e241f914e856ff9/u/a/ua-galaxy-a33-5g-a336-sm-a336blbgsek-531847005.jpg"
-            },
-            {
-                "model": "poco-X4-PRO-5G",
-                "picture": "https://mobile-review.com/all/wp-content/uploads/2022/03/2-4.jpg"
-            },
-            {
-                "model": "google-pixel-6-pro",
-                "picture": "https://img.mta.ua/image/cache/data/foto/z550/550960/photos/Google-Pixel-6-Pro-12128GB-Cloudy-White-60-Black-01-600x600.jpg"
-            },
-            {
-                "model": "samsung-galaxy-z-flip-3",
-                "picture": "https://img.mta.ua/image/cache/data/foto/z435/435897/photos/Samsung-Galaxy-Z-Flip-3-8128GB-Lavender-73-Green-01-600x600.jpg"
-            },
-            {
-                "model": "xiaomi-redmi-note-9-pro",
-                "picture": "https://img.mta.ua/image/cache/data/foto/z192/192360/photos/Xiaomi-Redmi-Note-9-Pro-01-600x600.jpg"
-            },
-            {
-                "model": "oneplus-9-pro",
-                "picture": "https://img.mta.ua/image/cache/data/foto/z323/323044/photos/OnePlus-9-Pro-8128GB-Morning-Mist-Silver-01-600x600.jpg"
-            },
-            {
-                "model": "google-pixel-5a",
-                "picture": "https://img.mta.ua/image/cache/data/foto/z475/475436/photos/38-01-600x600.jpg"
-            }
-        ]
-    }
-    return render(request, 'homepage.html', context)
+def list_view(request):
+    products = Product.objects.filter().order_by('category_id')
```

Comment:
Ти не використовуєш фільтр, отже більш правильним буде використання `all()`. Хоча це звісно призведе до проблем без пагінації, але на цей час так буде краще.

```suggestion
    products = Product.objects.all().order_by('category_id')
```